Client/Frontend/Browser/URLBarView.swift

```diff
@@ -379,30 +390,30 @@ class URLBarView: UIView {
     }
 
     func setAutocompleteSuggestion(suggestion: String?) {
-        locationView.editTextField.setAutocompleteSuggestion(suggestion)
+        locationTextField.setAutocompleteSuggestion(suggestion)
     }
 
-    func finishEditing() {
-        locationView.active = false
-        updateLayoutForEditing(editing: false)
-        delegate?.urlBarDidEndEditing(self)
+    func enterOverlayMode(locationText: String?) {
+        delegate?.urlBarDidEnterOverlayMode(self)
```

Comment:
From previous comments. Should this move to the end? Or even after the animation finishes?